R/dims.R

```diff
@@ -93,7 +97,7 @@ calc_umap <- function(fsce,
                       n_neighbors = 30,
                       min_dist = 0.3,
                       metric = "euclidean",
-                      seed = NA,
+                      seed = NULL,
```

Comment:
I believe this needs to be left as `NA`, as the `umap::umap` function expects either `NA` or an integer. (see `umap::umap.defaults`).